### PR TITLE
voxelands: 1512.00 -> 1704.00

### PR DESCRIPTION
**Pull request by prusnak**
###### Motivation for this change

Version update, Repo change URL change

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/games/voxelands/default.nix [hunk 1]
```diff
@@ -17,12 +34,24 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
-    cmake irrlicht libpng bzip2 libjpeg sqlite
-    libXxf86vm libGLU libGL openal libvorbis xlibsWrapper pkgconfig
+    bzip2
+    cmake
```
Comment (jonringer) [inline pkgs/games/voxelands/default.nix:38]: cmake and pkg-config should be in `nativeBuildInputs`

Comment (prusnak) [reply pkgs/games/voxelands/default.nix]: Addressed in 45f32f9e82b79d49c1b11e1c4630463cbde14168

**Review by doronbehar — APPROVED** [review]

**Comment by doronbehar** [comment]
It fails on aarch64 https://logs.nix.ci/?key=nixos/nixpkgs.104705&attempt_id=ce623944-0fb0-4184-8afb-3211576e4a57

Perhaps it should be marked as broken for this system?

**Comment by prusnak** [comment]
> Perhaps it should be marked as broken for this system?

This seems more like a issue with Irrlicht engine on aarch64 - see https://sourceforge.net/p/irrlicht/bugs/443/

So more like anything maybe we should mark irrlicht broken on aarch64 until this is resolved

**Comment by doronbehar** [comment]
> This seems more like a issue with Irrlicht engine on aarch64 - see [sourceforge.net/p/irrlicht/bugs/443](https://sourceforge.net/p/irrlicht/bugs/443/) (irrlich thinks the system libpng was compiled with NEON instructions support, but it seems it's not)

Is irrlicht a bundled dependency built within the build of voxeland? Because from the log it seems it's voxeland's build that fails: https://logs.nix.ci/?key=nixos/nixpkgs.104705&attempt_id=ce623944-0fb0-4184-8afb-3211576e4a57

**Comment by prusnak** [comment]
> Is irrlicht a bundled dependency built within the build of voxeland?

No. If I see it correctly voxeland uses system irrlicht.

**Comment by doronbehar** [comment]
> > Is irrlicht a bundled dependency built within the build of voxeland?
> 
> No. If I see it correctly voxeland uses system irrlicht.

Then if irrlicht was the failing build, the build log wouldn't have looked like this:

```
[ 99%] Linking CXX executable ../../bin/voxelands-server
[ 99%] Built target voxelands-server
[100%] Linking CXX executable ../../bin/voxelands
/nix/store/liq5wrz3yprc9623bb9lc90qv610gsd8-binutils-2.31.1/bin/ld: /nix/store/ci25cir0pkj8h2dmsaxppdcyqif2m0cb-irrlicht-1.8.4/lib/libIrrlicht.so: undefined reference to `png_init_filter_functions_neon'
collect2: error: ld returned 1 exit status
make[2]: *** [src/CMakeFiles/voxelands.dir/build.make:1370: ../bin/voxelands] Error 1
make[1]: *** [CMakeFiles/Makefile2:186: src/CMakeFiles/voxelands.dir/all] Error 2
make: *** [Makefile:171: all] Error 2
error: --- Error --- nix-build
error: --- Error --- nix-daemon
builder for '/nix/store/9lzw62zdgkzq8jb25xs9yq22ar793q81-voxelands-1704.00.drv' failed with exit code 2; last 10 log lines:
    414 |   for (u8 i = 0; i < 8; i++) {
        |   ^~~
  [ 99%] Linking CXX executable ../../bin/voxelands-server
  [ 99%] Built target voxelands-server
  [100%] Linking CXX executable ../../bin/voxelands
  /nix/store/liq5wrz3yprc9623bb9lc90qv610gsd8-binutils-2.31.1/bin/ld: /nix/store/ci25cir0pkj8h2dmsaxppdcyqif2m0cb-irrlicht-1.8.4/lib/libIrrlicht.so: undefined reference to `png_init_filter_functions_neon'
  collect2: error: ld returned 1 exit status
```

As you can see, it's voxelands' binaries that fail to link, not irrlicht's. Whatever the reason is, we should mark it as broken for aarch64, ideally with a comment saying the build expects neon symbols or something like that.

**Comment by prusnak** [comment]
> we should mark it as broken for aarch64

Addressed in 1ed49e3ea9201a6f932bcbe79e3cc1c32a5f9fc6

**Review by doronbehar — APPROVED** [review]

**Pull request merged**